sdk/ml/azure-ai-ml/azure/ai/ml/_schema/_sweep/search_space/normal.py

```diff
@@ -52,3 +52,9 @@ def predump(self, data, **kwargs):
         if not isinstance(data, (QNormal, QLogNormal)):
             raise ValidationError("Cannot dump non-QNormal or non-QLogNormal object into QNormalSchema")
         return data
+
+
+class IntegerQNormalSchema(QNormalSchema):
+    min_value = DumpableIntegerField(strict=True, required=True)
```

Comment:
Should these be mu, sigma instead of min_value/max_value ?

Reply:
Also, why do these need to be integers ? Would having q as Integer not suffice ?

Reply:
Same comment for quniform

Reply:
Thanks for catching this Phani. I updated the IntegerQNormalSchema. 

For the second question, we need all integers since quniform(1.5, 5.5, 1) => [1.5, 2.5, 3.5, 4.5, 5.5] which we donot want. The quantized values are equally probably starting from min_value to max_value.